FlowCrypt/src/main/java/com/flowcrypt/email/api/retrofit/response/model/MsgBlockFactory.kt

```diff
@@ -45,11 +43,19 @@ object MsgBlockFactory {
     return when (type) {
       MsgBlock.Type.PUBLIC_KEY -> {
         val keyDetails = if (content != null && complete) {
-          PgpKey.parseKeys(content).pgpKeyDetailsList.firstOrNull()
+          try {
+            PgpKey.parseKeys(content).pgpKeyDetailsList.firstOrNull()
+          } catch (e: Exception) {
+            e.printStackTrace()
+            null
+          }
```

Comment:
This may need a more clever error handling, in another milestone

Reply:
Agree. I will create an issue for that. For now, the current version just prevent any errors, but it should be improved